scipy/io/tests/test_mmio.py

```diff
@@ -28,6 +28,22 @@ def test_simple(self):
         b = mmread(fn)
         assert_array_almost_equal(a,b)
 
+    def test_simple_upper_triangle(self):
+        a = [[0,1],[0,0]]
+        fn = self.fn
```

Comment:
Any reason for this line in each test function, rather than `mmwrite(self.fn, a)`?